blog-posts/enterprises-benefit-from-scoped-npm-registries.md

```diff
@@ -51,9 +51,11 @@ Even if the registry is hosted internally, it is good practice to add authentica
 
 `username:password` ⇒ base64 encoded ⇒ `dXNlcm5hbWU6cGFzc3dvcmQ=`
 
-For encoding you can use the command line `echo -n 'my-string' | base64` (Linux) or an IDE (vscode with plugin) or search for an online base64 encoder.
+For encoding you could use the command line `echo -n 'my-string' | base64` (bash) or an IDE (e.g. vscode with plugin). Or alternatively you could even use a REPL (e.g. [JS REPL](https://nodejs.dev/learn/how-to-use-the-nodejs-repl) with the following command in JavaScript `Buffer.from('username:password', 'binary').toString('base64')`).
 
->💡 Use online encoder only for testing purposes. Never enter a productive username and/or password there.
+>💡 If you use the command-line, the password is saved in plaintext into the command-line history and can be retrieved from there. You should delete the password afterward (e.g. for bash you can delete the complete history with `history -c`)
```

Comment:
I've just learned that there is a way to bypass the history by adding a space in front of the command. On my machine that works but it seems to be configurable, so no guarantee that it works everywhere.

Generally I wouldn't advise people to clear their entire history with `history -c`. I would be absolutely lost and furious if that happened to me (over 10000 saved history items that make my daily workflow sooo much easier). I'd rather suggest to remove that single new entry that includes the password.

Here is a nice looking answer on stackexchange that I suggest to include. It also adds this trick to bypass the history by using a leading whitespace: https://unix.stackexchange.com/a/49216